src/Main.cpp

```diff
@@ -219,7 +219,7 @@ int main(int argc, char** argv) {
     s_tcpDispatchTask.start();
     s_uartMessageSender.start();
     s_tcpMessageSender.start();
-    s_spiMessageSender.start();
+    // s_spiMessageSender.start();
```

Comment:
Forgot to uncomment

Reply:
No it breaks if we don't have an ESP plugged in. So until we all work with a HiveSight, we will leave commented